ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -802,7 +802,7 @@ ethernet_interfaces:
     shape:
       rate: < "< rate > kbps" | "1-100 percent" | "< rate > pps" , supported options are platform dependent >
     qos:
-      trust: < dscp | cos >
+      trust: < dscp | cos | false >
```

Comment:
to avoid mixing types (string vs. boolean) and to also simplify the documentation part, we could use `trust: < dscp | cos | disabled >`.

Reply:
Ok, initial approach was to use the keyword **no** itself but it is converted to boolean again. So I used false. But I will change it to **disabled**